.github/workflows/l10n.yml

```diff
@@ -0,0 +1,27 @@
+name: daily l10n run
+
+on:
+  schedule:
+    - cron: "0 0 * * 1-5"
```

Comment:
Let's also enable this to be run manually using the `workflow_dispatch` event: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch